resources/js/pages/admin/dashboard/turnos/GestionTurnos.vue

```diff
@@ -0,0 +1,637 @@
+            });
+            this.spiner = false;
+            if (responseUpdate.data.errores.length > 0) {
+              console.warn('Errores al actualizar:', responseUpdate.data.errores);
+            }
+          }
+
+          if (eventosAEliminar.length > 0) {
+            const responseDelete = await axios.delete('/api/deleteCalendario', {
+              data: {
+                  calendarioIds: eventosAEliminar,
+              },
+            });
+            this.spiner = false;
+            if (responseDelete.data.no_encontrados.length > 0) {
+              console.warn('Errores al eliminar:', responseDelete.data.no_encontrados);
+            }
+          }
+          this.newEvents = [];
+          this.updatedEvents = [];
+          this.deletedEvents = [];
+          this.dataLoaded = true;
+          this.$toaster.success('Datos guardados correctamente');
+
+        } catch (error) {
+          this.spiner = false;
+          console.error('Ocurrio un error al guardar los datos:', error.response?.data || error.message);
+          this.$toaster.error('Ocurrio un error al guardar los datos.');
+          
+        }
+      },
+
+      validarDatos() {
+          this.$v.$touch();
+
+          if (this.$v.$invalid) {
+              this.$toaster.error("Hay errores en el formulario. Por favor, corrígelos.");
+              return false;
+          }
+          return true;
+      },
+
+      async getUsers() {
+          try {
+              const responseUsers = await axios.get('/api/getUsers');
+              this.users = responseUsers.data;
+          } catch (errors) {
+              console.log(errors.response.data.errors)
+          }
+      },      
+      async getSedesByUser(userId) {        
+          try {
+              const response = await axios.get('/api/getSedesAndClientesByUser', {
+                  params: {
+                      user_id: userId,
+                  },
+              });
+              this.sedesByUser =  response.data.sedes;
+              this.sedesByUser = response.data.sedes.map(sede => ({
+                id: sede.sede_id,
+                name: sede.sede_nombre,
+              }));
+          } catch (error) {
+              console.error('Error al obtener sedes:', error.response?.data?.errors || error.message);
+              this.$toaster.error('Hubo un error al obtener sedes.');
+          }
+      },
+
+      async handleUserSelection(userId) {
+        await this.getSedesByUser(userId);
+      },      
+
+      viewDay({ date }) {
+        const selectedDate = new Date(date + 'T00:00:00');
+        const today = new Date();
+        today.setHours(0, 0, 0, 0);
+        selectedDate.setHours(0, 0, 0, 0);
+        if (selectedDate < today) {
+          return;
+        }
+        this.focus = date;
+        this.type = 'day';
+        this.newEvent.startDate = date;
+        this.newEvent.endDate = date;
+      },
+  
+      setDate(date) {
```

Comment:
```suggestion
       setDate(date) {
        this.dateMenu = false; 
      },
  
      setEndDate(date) {
        this.dateMenuEnd = false;
      },
```
Se está utilizando el date?